### PR TITLE
show sampled uris from a topic

### PR DESCRIPTION
**Pull request by yingjieMiao**
@stkem 

mostly admin work

### CODE REVIEW

File: kifi-backend/modules/cortex/app/com/keepit/cortex/dbmodel/URILDATopicRepo.scala [hunk 1]
```diff
@@ -107,4 +108,11 @@ class URILDATopicRepoImpl @Inject() (
     query.as[(Int, Int)].list map { case (topic, count) => (LDATopic(topic), count) }
 
   }
+
+  def getLatestURIsInTopic(topicId: LDATopic, version: ModelVersion[DenseLDA], limit: Int)(implicit session: RSession): Seq[Id[NormalizedURI]] = {
+    import StaticQuery.interpolation
+
+    val q = sql"select uri_id from uri_lda_topic where first_topic = ${topicId.index} and version = ${version.version} order by updated_at desc limit ${limit}"
```
Comment (stkem) [inline kifi-backend/modules/cortex/app/com/keepit/cortex/dbmodel/URILDATopicRepo.scala:115]: why not use slick here?

Comment (yingjieMiao) [inline kifi-backend/modules/cortex/app/com/keepit/cortex/dbmodel/URILDATopicRepo.scala:115]: this seems to be more straightforward, and tested.

Comment (stkem) [inline kifi-backend/modules/cortex/app/com/keepit/cortex/dbmodel/URILDATopicRepo.scala:115]: ok

**Comment by stkem** [comment]
lgtm

File: kifi-backend/modules/shoebox/app/views/admin/ldaDetail.scala.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  <table class = "table table-bordered">
+  <tr>
+    <th>uriId</th>
+    <th>Title</th>
+  </tr>
+
+  @data.sampleURIs.map{ uri =>
+    <tr>
+      <td>@uri.id.get</td>
+      <td><a href = "@{com.keepit.controllers.admin.routes.ScraperAdminController.getScraped(uri.id.get)}">@uri.title.getOrElse("No Title")</a></td>
```
Comment (eishay) [inline kifi-backend/modules/shoebox/app/views/admin/ldaDetail.scala.html:15]: great

**Pull request merged**